production/simple-scalable/docker-compose.yaml

```diff
@@ -0,0 +1,175 @@
+---
+version: "3"
+
+networks:
+  loki:
+
+services:
+  read:
+    image: grafana/loki:2.4.1
+    command: "-config.file=/etc/loki/config.yaml -target=read"
+    ports:
+      - 3100
+      - 7946
+      - 9095
+    volumes:
+      - ./loki-config.yaml:/etc/loki/config.yaml
+    depends_on:
+      - minio
+    networks: &loki-dns
+      loki:
+        aliases:
+          - loki
+
+  write:
+    image: grafana/loki:2.4.1
+    command: "-config.file=/etc/loki/config.yaml -target=write"
+    ports:
+      - 3100
+      - 7946
+      - 9095
+    volumes:
+      - ./loki-config.yaml:/etc/loki/config.yaml
+    depends_on:
+      - minio
+    networks:
+      <<: *loki-dns
+
+  promtail:
+    image: grafana/promtail:2.4.1
+    logging: &logging
+      driver: loki-compose
```

Comment:
I'm guessing this is the loki-driver, I think people can get confused by not having it installed so we might want to add a point in the documentation.

Reply:
oh good point! will do.